test/edu/csus/ecs/pc2/imports/ccs/ContestSnakeYAMLLoaderTest.java

```diff
@@ -3500,6 +3501,118 @@ public void testScoreboardHTMLLocations() throws Exception {
 
     }
     
+    public void testLoadDisplay() throws Exception {
+        
+        String dirname = getOutputDataDirectory(getName());
+        ensureDirectory(dirname);
+        
+        String[] yamlLines = { //
+                "",
+                "team-display-on-scoreboard-string : '{:teamname}'", //
+                "",
+                "accounts:", //
+                "  - account: TEAM", //
+                "    site: 1", //
+                "    count: 22", //
+        };
+
+
+        IInternalContest contest = loader.fromYaml(null, yamlLines, dirname);
+        
+        Account[] teams = getTeamAccounts(contest);
+        assertEquals("Team count", 22, teams.length);
+
+        String displayString = contest.getContestInformation().getTeamDisplayOnScoreboard();
+        assertEquals("Team getTeamDisplayOnScoreboard", "{:teamname}", displayString);
+        
+        /**
+         * 
+     * Client/Team number - {:clientnumber} = 514
+     * Country Code - {:countrycode} = CAN
+     * CMS/External ID - {:externalid} = 309407
+     * Group Id Number - {:groupid} = 12545
+     * Group name - {:groupname} = Canada - University of British Columbia D1
+     * Long School name - {:longschoolname} = UBC! (U British Columbia)
+     * Short Shool name - {:shortschoolname} = U British Columbia
```

Comment:
Same comments as on a previous file:  these are not accurate samples.  See the earlier comment for details.

Reply:
Just a note that my understanding is that you don't disagree that the examples are not well-constructed, but rather that you're choosing to "resolve" the conversation without making any changes because you don't want to spend time fixing JavaDoc.   I'm ok with that.  However, as someone (I think it was YOU) told me recently, "If the code disagrees with the comments, both are wrong".   I would say that also sort of applies here.  But so be it.